content/events/19/info.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "جلسه ۱۹ "
+date: 2024-05-23T14:30:00.000Z
+layout: event
+type: event
+eventNumber: 18
+eventDate: 1403-03-03
+startTime: 18:00
+endTime: 20:00
+address: "خیابان غفاری، پارک علم و فناوری"
+registerLink: https://evand.com/events/birjandlug-18
+speakers: [سعید آبادیان]
+topics: [Linux-Essentials_p2]
+lat: 32.845253
+lng: 59.229300
+locations: [پارک علم و فناوری]
+---
+
+### لینک‌های مرتبط
+
+ارائه «لینوکس مقدماتی» (سعید آبادیان)
+
+- [Linux Essentials](https://learning.lpi.org/pdfstore/LPI-Learning-Material-010-160-de.pdf)
```

Comment:
it would be good to add the titles you mentioned in your pull request message here as well.